flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/MapJoinLeftRunner.scala

```diff
@@ -31,7 +32,19 @@ class MapJoinLeftRunner[IN1, IN2, OUT](
   override def flatMap(multiInput: IN1, out: Collector[OUT]): Unit = {
     broadcastSet match {
       case Some(singleInput) => function.join(multiInput, singleInput, out)
-      case None =>
+      case None => {
```

Comment:
@fhueske, If I use your code in  `MapJoinLeftRunner ` I get following spooky error:
```
scala.MatchError: None (of class scala.None$)

	at org.apache.flink.table.runtime.MapJoinLeftRunner.flatMap(MapJoinLeftRunner.scala:34)
	at org.apache.flink.api.common.operators.base.FlatMapOperatorBase.executeOnCollections(FlatMapOperatorBase.java:73)
```
I have googled this issue and find out that https://groups.google.com/forum/#!topic/akka-user/jvbbilXDKdA it could be related to akka.
So, I left code in `MapJoinLeftRunner` as was